src/battle_script_commands.c

```diff
@@ -8394,14 +8397,43 @@ static void Cmd_returntoball(void)
     gBattlescriptCurrInstr = cmd->nextInstr;
 }
 
+static void LearnCombinedNewMoves(u32 monId, u16 *learnMove)
```

Comment:
Is there a reason this is A) a separate function and B) a function with ``CMD_ARGS``?

Reply:
[`a180eac` (#4901)](https://github.com/rh-hideout/pokeemerald-expansion/pull/4901/commits/a180eacf117f7faea3ad94c46acdb336c9861c72)